mlrun/config.py

```diff
@@ -134,6 +134,9 @@
         "resources": {
             "delete_crd_resources_timeout": "5 minutes",
         },
+        "alert_activations": {
+            "retention_days": 14 * 7,  # days
```

Comment:
redundant, you can say "14 weeks" if you want to explain, but `days` is already written in the key 
```suggestion
            "retention_days": 14 * 7, 
```